1.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * @param {number[]} nums
+ * @param {number} target
+ * @return {number[]}
+ */
+var twoSum = function(nums, target) {
+    for(let i=0; i<nums.length; i++){
+        for(let j=i+1;j<nums.length;j++){
+            if(nums[i]+nums[j]==target){
```

Comment:
```suggestion
    for(let i = 0; i < nums.length; i++){
        for(let j = i + 1; j < nums.length; j++){
            if(nums[i] + nums[j] == target){
```

Reply:
You should use the strict equality operator `===` instead of the loose equality operator `==` unless there is a specific reason to use the loose one.